dev_requirements.txt

```diff
@@ -11,7 +11,7 @@ flaky
 freezegun==0.3.9
 ipdb
 mock>=1.3.0
-mypy==0.782
+mypy==0.950
```

Comment:
Mypy would not pass the way the code was written. Why? MyPy has a delineation between `Type Variables` and `TypeAlias`s. To force dynamic classes such as `Relation` to be read as `TypeAlias`s , we need a newer version of MyPy for use of `type_extensions`.

If anyone goes digging, they'll find that eventually `typing` gets `TypeAlias` as a native member in later versions of Python, but this doesn't include all `dbt`s supported versions of Python. This solution was selected as the more surgical approach.